src/metabase/models/permissions.clj

```diff
@@ -1037,10 +1036,10 @@
    (t2/delete! Permissions :group_id (u/the-id group-or-id) :object [:like "/query/%"])
    (t2/delete! Permissions :group_id (u/the-id group-or-id) :object [:like "/data/%"])
    (try
-     (db/insert-many! Permissions
-       (map (fn [path-object]
-              {:group_id (u/the-id group-or-id) :object path-object})
-            (distinct (conj (->v2-path path) path))))
+     (t2/insert! Permissions
```

Comment:
shouldn't this be an `insert-returning-instances!`?

Reply:
yea good point, we should have these insert methods in model ns to return the instances.

Reply:
changed in abbea87184